decentraland/unity-explorer
# Fix: footsteps and walked distance badge

## What does this PR change?
Fix #2189 

Fixed walked distance analytics. 
- added sending on intervals and on amount of steps reached. Accumulation will be done on the backend
- added steps count to the debug panel
- handle the case of re-login with the different account

## How to test the changes?

1. Launch the explorer
2. Open Debug panel for Badges
3. Check that step count is increasing for about 30 seconds and then become 0, and increase again. 
4. Accumulate some steps and re-login -> observe steps counter to be zero

Additionally steps sound is back. However, jumping on place is still not producing the sound

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
